apps/lib/apps.c

```diff
@@ -2106,29 +2106,37 @@ unsigned char *next_protos_parse(size_t *outlen, const char *in)
     return out;
 }
 
-void print_cert_checks(BIO *bio, X509 *x,
+int X509_check_cert(BIO *bio, X509 *x,
                        const char *checkhost,
-                       const char *checkemail, const char *checkip)
+                       const char *checkemail, const char *checkip, int print)
 {
+    int ret = 1;
+
     if (x == NULL)
-        return;
+        return 0;
+
     if (checkhost) {
-        BIO_printf(bio, "Hostname %s does%s match certificate\n",
-                   checkhost,
-                   X509_check_host(x, checkhost, 0, 0, NULL) == 1
-                       ? "" : " NOT");
+	    int valid_host = X509_check_host(x, checkhost, 0, 0, NULL);
```

Comment:
nit: project style is 4-spaces indentation

Reply:
fixed in last commit.